Cargo.toml

```diff
@@ -10,6 +10,7 @@ links = "mysqlclient"
 
 [build-dependencies]
 pkg-config = "0.3.9"
+bindgen = "0.69.4"
```

Comment:
This change still has a hard dependency on bindgen. I rather prefer putting this behind feature flag that is disabled by default.

Something like this: https://github.com/sgrif/pq-sys/blob/master/Cargo.toml#L29